### ISSUE TITLE
NuGet Autocomplete for Multiple NuGet sources

### ISSUE THREAD

**Issue by marzullo** [roslynpad/roslynpad#342]
**Describe the solution you'd like**
1. Packages are currently only returned for one source -- the autocomplete should return packages for all sources 
2. The search API for NuGet seems to return packages that are unrelated to the search term being used (information such as icon names for packages is matched). So an additional filter can be added to match the search result set on the NuGet package Identity.Id field so these completely unrelated packages are not returned in the autocomplete list.

**Additional context**
Work done here, very small:
https://github.com/marzullo/RoslynPad/commit/3d730ee2145d9b365c29b082423cba688514f764

Is it okay if I submit a PR for these changes? Any comments?

**Comment by aelij** [comment]
Hi. Sure you're welcome to submit a PR - but can you explain why you added

```cs
result.Where(x => x.Identity.Id.Contains(searchTerm))
```

The NuGet search engine already filters using this string in `SearchAsync`.

**Comment by marzullo** [comment]
Right, it seems that the `SearchAsync` is filtering from the string, but it looks like to me that it is matching on any metadata with the `searchTerm` instead of just the package name itself. I can provide an example of this happening where a bunch of weird packages are returned because there's a match in the metadata, if you want to see it. I guess I just find this a little annoying since I think from a UX perspective that you would typically only be searching by package name, but it's not a necessary change.

**Comment by aelij** [comment]
I would like to keep the current functionality, so please don't include this extra filter in your PR :)
